jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -401,4 +406,33 @@ private static Boolean getPropertyAsBoolean(String name) {
     return null;
   }
 
+  private static Tracer.Builder initTracerTags(Tracer.Builder builder) {
```

Comment:
per comment above, I'd rename to `tagsFromEnv` and return a Map (don't need builder at all).